languages/csharp/exercises/concept/overflow/.docs/after.md

```diff
@@ -0,0 +1,48 @@
+The exercise shows the behavior of various numeric types when they overflow, i.e. when their capacity is insufficient to contain the value resulting from a computation such as an arithmetic operation or cast.
+
+- unsigned [integers][integral-numeric-types] (`byte`, `ushort`, `uint`, `ulong`) will wrap around to zero (the type's maximum value + 1 acts as a modulus) unless [broadly speaking][checked-compiler-setting] they appear within a [`checked`][checked-and-unchecked] block in which case an instance of `OverflowException` is thrown. `int` and `long` will behave similarly except that they wrap around to `int.MinValue` and `long.minValue` respectively.
+
+```csharp
+var xx = new Random().Next();
+checked
+{
+    int expr = int.MaxValue * xx;  // overflow exception is thrown
+}
+
+// or
+
+int expr2 = checked(int.MaxValue * xx);  // overflow exception is thrown
```

Comment:
We could get rid of the `Random` bit by introducing a variable, with which we can increment by one to hopefully even more clearly show the overflow:

```suggestion
int one = 1;
checked
{
    int expr = int.MaxValue + one;  // overflow exception is thrown
}

// or

int expr2 = checked(int.MaxValue + one);  // overflow exception is thrown
```